org.mozilla.FirefoxNightly/org.mozilla.FirefoxNightly.json

```diff
@@ -5,6 +5,16 @@
     "sdk": "org.gnome.Sdk",
     "command": "firefox",
     /*"tags": [""],*/
+    "add-extensions": {
+        "org.mozilla.FirefoxNightly.flashplayer": {
+	    "directory": "lib/mozilla/plugins",
+	    "add-ld-path": "lib",
+	    "merge-dirs": "lib/mozilla/plugins",
+	    "subdirectories": true,
```

Comment:
If we're providing just one Flash extension point, we wouldn't need merge/subdirectories.